packages/fxa-content-server/app/tests/spec/views/support.js

```diff
@@ -212,6 +212,36 @@ describe('views/support', function () {
         });
     });
 
+    it('uses the SKU in metrics when given a Google Play Subscription', () => {
```

Comment:
Nit: Maybe note that the SKU is used for the `planId` in particular?

Reply:
In theory I agree but practically it's not worth the cost of running CI all over again.

Reply:
Oh, I guess I'll be updating the commit message anyway, so sure, I can update this message along with that.